src/engine/source/indexerconnector/include/indexerConnector.hpp

```diff
@@ -50,21 +50,13 @@ class EXPORTED IndexerConnector final
      * @brief Class constructor that initializes the publisher.
      *
      * @param config Indexer configuration, including database_path and servers.
-     * @param logFunction Callback function to be called when trying to log a message.
      * @param timeout Server selector time interval.
      * @param workingThreads Number of working threads used by the dispatcher. More than one results in an unordered
      * processing.
      */
     explicit IndexerConnector(const nlohmann::json& config,
-                              const std::function<void(const int,
-                                                       const std::string&,
-                                                       const std::string&,
-                                                       const int,
-                                                       const std::string&,
-                                                       const std::string&,
-                                                       va_list)>& logFunction = {},
                               const uint32_t& timeout = DEFAULT_INTERVAL,
-                              const uint8_t workingThreads = 1);
+                              uint8_t workingThreads = 1);
```

Comment:
The const qualifiers was removed from here but not on the Cpp file. It was on purpose to remove it?

Reply:
const-qualification of parameters only has an effect in function definitions